purl-dev/schema.xml

```diff
@@ -14,6 +14,14 @@
 
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
     <field name="timestamp" type="date" indexed="true" stored="true" default="NOW" multiValued="false"/>
+    <copyField source="*_s" dest="*_ssim"/>
+    <copyField source="*_t" dest="*_tsim"/>
+  
+    <!-- Temporary Dynamic Fields that mimic the Argo schema to allow Purl-Fetcher fields to Match Dor-Fetcher fields -->
+     <!-- Following the upgrade of Argo-prod these will be delated and _ssim/_tsim will be used -->
+    <dynamicField name="*_s"  type="string"  indexed="true"  stored="true" multiValued="true"/>
```

Comment:
Ah, I assumed the code was going the other way (using some legacy indexing code). How tricky would it be to make those fields independently configurable in your code? It seems like having the flexibility to version argo and purl schemas independently is probably valuable, isn't it?

Otherwise, I'd suggest limit the `copyField` to just the fields you care about, with a great big note about why we're doing this, in order to limit future mistakes or unexpected behavior (say, your code begins depending on some other `_s` field.. )